opal-core-ws/src/main/java/org/obiba/opal/web/project/ProjectResource.java

```diff
@@ -134,13 +134,6 @@ public KeyStoreResource getKeyStoreResource() {
   public VCFStoreResource getVCFStoreResource() {
     Project project = getProject();
     if (!opalRuntime.hasServicePlugins()) throw new NoSuchServiceException(VCFStoreService.SERVICE_TYPE);
-    if (!project.hasVCFStoreService()) {
-      // for now get the first one. Some day, the service type will be a project admin choice
-      ServicePlugin service = opalRuntime.getServicePlugins()
-          .stream().filter(s -> opalRuntime.isVCFStorePluginService(s)).iterator().next();
-      project.setVCFStoreService(service.getName());
-      projectService.save(project);
-    }
```

Comment:
For now, yes, but couldn't there be more than one vcf store plugin in the future?